WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -39,6 +39,7 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
     private let productUIImageLoader: ProductUIImageLoader
 
     private let currency: String
+    private let titleString: String // Navigation title
```

Comment:
We tend to add documentation like:

```suggestion
    /// Navigation title
    ///
    private let titleString: String 
```

That way is picked up by Xcode and displayed on the help menu

Reply:
I will remember this, thanks.